pynicotine/plugins/core_commands/__init__.py

```diff
@@ -26,39 +26,637 @@ def __init__(self, *args, **kwargs):
+            },
+            "pm": {
+                "callback": self.pm_chat_command,
+                "description": _("Open private chat window for user"),
+                "disable": ["cli"],
+                "group": _("Private Chat"),
+                "usage": ["<user>"]
+            },
+            "say": {
+                "callback": self.say_chat_command,
+                "description": _("Say message in specified chat room"),
+                "disable": ["cli"],
+                "group": _("Chat Rooms"),
+                "usage": ["<room>", "<message..>"]
+            },
             "close": {
-                "description": "Close private chat",
                 "aliases": ["c"],
-                "disable": ["cli"],
-                "group": "Private Chat",
                 "callback": self.close_command,
-                "usage_chatroom": ["<user>"],
+                "description": _("Close private chat"),
+                "disable": ["cli"],
+                "group": _("Private Chat"),
+                "usage": ["<user>"],
+                "usage_private_chat": ["[user]"]
+            },
+            "leave": {
+                "aliases": ["l"],
+                "callback": self.leave_command,
+                "description": _("Leave chat room"),
+                "disable": ["cli"],
+                "group": _("Chat Rooms"),
+                "usage": ["<room>"],
+                "usage_chatroom": ["[room]"]
+            },
+            "now": {
+                "callback": self.now_playing_command,
+                "description": _("Display the Now Playing script's output"),
+                # "disable": ["cli"],
+                "group": _("Now Playing"),
+            },
+            "add": {
+                "aliases": ["buddy"],
+                "callback": self.add_buddy_command,
+                "description": _("Add user to buddy list"),
+                "group": _("Users"),
+                "usage": ["<user>"],
+                "usage_private_chat": ["[user]"]
+            },
+            "rem": {
+                "aliases": ["unbuddy"],
+                "callback": self.remove_buddy_command,
+                "description": _("Remove user from buddy list"),
+                "group": _("Users"),
+                "usage": ["<buddy>"],
+                "usage_private_chat": ["[buddy]"]
+            },
+            "ban": {
+                "callback": self.ban_user_command,
+                "description": _("Stop connections from user"),
+                "group": _("Users"),
+                "usage": ["<user>"],
+                "usage_private_chat": ["[user]"]
+            },
+            "unban": {
+                "callback": self.unban_user_command,
+                "description": _("Remove user from ban list"),
+                "group": _("Users"),
+                "usage": ["<user>"],
+                "usage_private_chat": ["[user]"]
+            },
+            "block": {  # new
```

Comment:
Suggest ban-ip

Reply:
This command does not (currently) support accepting an IP address as argument, so the proposed name would be misleading.